sdk/assemblyscript/src/assembly/database.ts

```diff
@@ -256,38 +245,27 @@ export class Location {
   // This is used in lieu of the @json decorator, so that the class can be
   // serialized to a string in SQL format.
 
-  __INITIALIZE(): this {
-    return this;
+  @serializer
+  serialize(self: Location): string {
+    return self.toString();
```

Comment:
I think there's a bug here that was here previously, since the resulting string should be wrapped in quotes after being serialized.

Reply:
@mattjohnsonpint, do I have it correct that quotes need to be added to the output like:
```suggestion
    return '"' + self.toString() + '"';
```

Reply:
Yes.  Otherwise it's not valid JSON.